### ISSUE TITLE
Display report status changes and their motivations on report page.

### ISSUE THREAD

**Issue by evilaliv3** [globaleaks/whistleblowing-software#4115]
### Proposal

This ticket is dedicated to the analysis and implementation for a functionality to show information about statuses changes and their motivations on the report page.

The idea is to extend the "Comment" section adding a comment for each report showing the following information
Author: The user that reported
Date: The date of the operation
Text: "Status: $StatusBeforeChange -> $StatusAfterChange

Examples:
```
Giorgio Fraschini
Date: June 12, 2024
Status: Opened

Giorgio Fraschini
Date: June 12, 2024
Status: Close
Motivation: Case handled with complete whistleblower's protection
```

### Motivation and context
Users should be able to see a track of the changes of status of reports and their motivations

**Comment by evilaliv3** [comment]
@msmannan00 has produced a vary simple and nice patch that implements the feature.

Here is a screenshot of the current implementation:
![Screenshot from 2024-06-24 14-53-35](https://github.com/globaleaks/GlobaLeaks/assets/217034/bf18a4d8-13d1-4d58-b514-270b955b545d)

I suggest we could not open a pull request and collect some community feedback and then proceed merging it.

**Comment by evilaliv3** [comment]
Closing as completed and released in 5.0.0

**Comment by evilaliv3** [comment]
This was actually removed considering users could provide a motivation simply using the comments before closing the case.
